.eslintrc.js

```diff
@@ -53,12 +53,14 @@ module.exports = {
         "import/no-duplicates": "off",
         "import/no-mutable-exports": "off",
         "import/no-unresolved": "off",
-
         // Temporarily work around a bug in eslint-plugin-svelte3.
         //
         // https://github.com/sveltejs/eslint-plugin-svelte3/issues/41#issuecomment-572503966
         "no-multiple-empty-lines": ["error", { max: 2, maxBOF: 2, maxEOF: 0 }],
       },
+      settings: {
+        "svelte3/ignore-styles": () => true,
```

Comment:
Can you add a comment on why we need to ignore styles? (I assume it's because we're now using sass tokens)